### DISCUSSION TITLE
tvOS 18?

### DISCUSSION THREAD

**Discussion by mateverywhere** [maxileith/homebridge-appletv-enhanced#509] [General]
Hi all,
Just curious if anyone has this plugin working successfully with tvOS 18 beta? If so, any glaring bugs to be aware of?

**Comment by maxileith** [comment]
When adding a new Apple TV device to the home app, inputs will be named

1. Input 1
2. Input 2
3. Input 3
4. ...

Instead of the actual app names.

**Reply by pmoney303** [reply]
Will there be a fix for that, or a workaround where we can ID which app represents each number and manually update it?

**Reply by maxileith** [reply]
Please refer to https://github.com/maxileith/homebridge-appletv-enhanced/issues/627

In short: this is a bug of TvOS 18 ... so Apple has to fix that. However, I have implemented a workaround that resets the names after a reboot of the plugin if the plugin detects that the names are faulty.
